### ISSUE TITLE
Snap releases only include bash-completions

### ISSUE THREAD

**Issue by dc-mak** [twpayne/chezmoi#660]
## Describe the bug

Snap releases only include bash-completions.

## To reproduce

Install via snap, look in /snap/chezmoi/current/completions

## Expected behavior

It should include all supported completions (zsh, fish)

**Comment by twpayne** [comment]
Totally agree that this is a bug.

Right now, snap only supports bash completions, as per the [snap documentation on completions](https://snapcraft.io/docs/tab-completion-for-snaps).

I did a [quick search of the snapcraft forum](https://forum.snapcraft.io/search?q=zsh%20completion) which only turned up a few hits, the most relevant seemingly [this question about zsh](https://forum.snapcraft.io/t/zsh-integration-for-the-snap-command/11094), and nothing to indicate that the situation has changed for now. [Searching for fish completion on the snapcraft forum](https://forum.snapcraft.io/search?q=fish%20completion) didn't find anything.

So, basically snapcraft needs to support non-bash completions so that chezmoi can include them.

**Comment by twpayne** [comment]
I asked on the Snapcraft forums about the status of this: https://forum.snapcraft.io/t/status-of-completions-for-shells-other-than-bash-e-g-zsh-fish/16482

**Comment by twpayne** [comment]
For info, there has been no response to the question I posted on the Snapcraft forums so sadly I suspect that this won't be fixed for a while.

**Comment by dc-mak** [comment]
No worries, thank you very much for this tool regardless! Very kind of you

**Comment by twpayne** [comment]
There's been no update from the Snapcraft developers on this issue for two years, so sadly I'm going to close this as `won't fix`. I hope that someday Snapcraft will support shell completions for shells other than bash, at which point we can re-open this issue.
